test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:12
+
+ENV password=notpassword1
+ENV dbpass=notpassword2
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High <!-- 4d68a571-3fff-4276-9aee-adf07444bd2f -->